frontend/src/metabase/admin/tasks/containers/BugReportApp.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import { Box } from "grid-styled";
+import AdminHeader from "metabase/components/AdminHeader";
+import { t } from "ttag";
+import { UtilApi } from "metabase/services";
+
+function navigatorInfo() {
+  return {
+    appVersion: navigator.appVersion,
+    language: navigator.language,
+    platform: navigator.platform,
+    userAgent: navigator.userAgent,
+    vendor: navigator.vendor
+  };
+}
+
+export default class BugReportApp extends Component {
+  constructor() {
+    super();
+    this.state = {
```

Comment:
FYI if all you're doing in a constructor is setting `state` (and it doesn't depend on `props`) you can replace the constructor with:

```
state = {
   details: {}
}
```